arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -267,5 +268,199 @@ public virtual async Task<GetCollectionFiguresResponse> GetCollectionFiguresAsyn
+            }
+        }
+
+        /// <summary>
+        /// Load Indexes into Memory.
+        /// Caches all index entries of this collection into the main memory. 
+        /// Therefore it iterates over all indexes of the collection and 
+        /// stores the indexed values, not the entire document data, 
+        /// in memory.
+        /// PUT /_api/collection/{collection-name}/loadIndexesIntoMemory
+        /// </summary>
+        /// <param name="collectionName">Name of the collection.</param>
+        /// <returns></returns>
+        public virtual async Task<LoadIndexesIntoMemoryResponse> LoadIndexesIntoMemoryAsync(string collectionName)
+        {
+            if (string.IsNullOrWhiteSpace(collectionName))
+            {
+                throw new System.ArgumentNullException("collectionName", "collectionName is required");
+            }
+            string uriString = _collectionApiPath + "/" + WebUtility.UrlEncode(collectionName) + "/loadIndexesIntoMemory";
+            using (var response = await _transport.PutAsync(uriString, new byte[] { }).ConfigureAwait(false))
+            {
+                if (response.IsSuccessStatusCode)
+                {
+                    var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
+                    return DeserializeJsonFromStream<LoadIndexesIntoMemoryResponse>(stream);
+                }
+                throw await GetApiErrorException(response).ConfigureAwait(false);
+            }
+        }
+
+        /// <summary>
+        /// Recalculates the document count of a collection.        
+        /// PUT /_api/collection/{collection-name}/recalculateCount
+        /// </summary>
+        /// <param name="collectionName">Name of the collection.</param>
+        /// <returns></returns>
+        public virtual async Task<RecalculateCountResponse> RecalculateCountAsync(string collectionName)
+        {
+            if (string.IsNullOrWhiteSpace(collectionName))
+            {
+                throw new System.ArgumentNullException("collectionName", "collectionName is required");
+            }
+            string uriString = _collectionApiPath + "/" + WebUtility.UrlEncode(collectionName) + "/recalculateCount";
+            using (var response = await _transport.PutAsync(uriString, new byte[] { }).ConfigureAwait(false))
+            {
+                if (response.IsSuccessStatusCode)
+                {
+                    var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
+                    return DeserializeJsonFromStream<RecalculateCountResponse>(stream);
+                }
+                throw await GetApiErrorException(response).ConfigureAwait(false);
+            }
+        }
+
+        /// <summary>
+        /// Returns the responsible shard for a document.        
+        /// PUT /_api/collection/{collection-name}/responsibleShard
+        /// </summary>
```

Comment:
A minor thing: it might be worth adding a `remarks` that this is only available for clusters, as mentioned in the online doc. Example:

```
<remarks>
This method is only available in a cluster.
</remarks>
```

https://www.arangodb.com/docs/stable/http/collection-getting.html#return-responsible-shard-for-a-document

Or add it to the summary like for `/shards`, it should be consistent in any case.